packages/serverpod/lib/src/database/database_query.dart

```diff
@@ -41,27 +44,72 @@ class SelectQueryBuilder {
       orderBy: _orderBy,
       where: _where,
     );
-
     var selectColumns = [..._fields, ..._gatherIncludeColumns(_include)];
 
     var subQueries = _buildSubQueries(orderBy: _orderBy);
     var select = _buildSelectStatement(selectColumns);
     var join =
         _buildJoinQuery(where: _where, orderBy: _orderBy, include: _include);
     var groupBy = _buildGroupByQuery(selectColumns, orderBy: _orderBy);
-    var where = _buildWhereQuery(where: _where);
-    var orderBy = _buildOrderByQuery(orderBy: _orderBy);
+    var where = _buildWhereQuery(
+      where: _where,
+      listQueryAdditions: _listQueryAdditions,
+    );
 
     var query = '';
-    if (subQueries != null) query += '$subQueries ';
     query += 'SELECT $select';
     query += ' FROM "${_table.tableName}"';
     if (join != null) query += ' $join';
     if (where != null) query += ' WHERE $where';
-    if (groupBy != null) query += ' GROUP BY $groupBy';
-    if (orderBy != null) query += ' ORDER BY $orderBy';
-    if (_limit != null) query += ' LIMIT $_limit';
-    if (_offset != null) query += ' OFFSET $_offset';
+
+    if (groupBy != null) query += ' $groupBy';
+    if (_orderBy != null) {
+      query +=
+          ' ORDER BY ${_orderBy?.map((order) => order.toString()).join(', ')}';
+    }
```

Comment:
Question: Any specific reason for not build the order by and group by queries like we did before? Feels like the old diff reads better and gives an overview what parts we are currently building for the SQL code.